crates/kitsune_p2p/kitsune_p2p/Cargo.toml

```
@@ -82,7 +83,8 @@ itertools = "0.12"
 workspace = true
 
 [features]
-default = ["tx2", "tx5"]
+
+default = ["tx5"]
```

```suggestion
default = ["dep:tx5"]
```

But, also: do we want this to be an optional features? Kitsune doesn't work without it.

---

What does `dep:` do?

---

It distinguishes between features that are optional dependencies (`dep:`) and non-dependency features added by the crate dev.

---

Actually, I removed all use of the tx5 feature in this PR, since there is no alternative. So I think the change should actually be:

```suggestion
default = []
```

---

Perhaps instead we should just make the tx5 dep non-optional, and make the default feature list empty?

---

That was my suggestion at the top of this thread.

---

Ah, I get it now. Yes I'll make the dep non-optional